src/vivarium_public_health/risks/data_transformation.py

```diff
@@ -90,3 +90,22 @@ def compute_paf(g):
     paf = paf.replace(-np.inf, 0)  # Rows with zero exposure.
 
     return paf
+
+
+def split_risk_from_type(full_risk: str):
+    """Expecting risk to specified as type.name (where type is singular).
+    Splitting out type and name."""
+    split = full_risk.split('.')
+    if len(split) != 2:
+        raise ValueError(f"You must specify the risk as <risk_type.risk_name>. You specified {full_risk}.")
```

Comment:
Probably give them quote marks instead of the angle brackets.  Otherwise people will put in angle brackets and get confused.